src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/RepaymentPlanContentProviderTest.java

```diff
@@ -19,6 +19,7 @@ public void createImmediatePaymentOption() {
             .withPaymentOption(PaymentOption.IMMEDIATELY)
             .build();
 
+
```

Comment:
remove this line please, too much space

Reply:
ok :), but i haven't finished working on tests yet "[WIP - Tests Pending] ROC-3258: Judgement Request payment by instalment"